Source/WebCore/platform/Decimal.cpp

```diff
@@ -48,7 +48,7 @@ static int const ExponentMax = 1023;
 static int const ExponentMin = -1023;
 static int const Precision = 18;
 
-static const uint64_t MaxCoefficient = UINT64_C(0x16345785D89FFFF); // 999999999999999999 == 18 9's
+static const uint64_t MaxCoefficient = UINT64_C(0xDE0B6B3A763FFFF); // 999999999999999999 == 18 9's
```

Comment:
I wonder if this mistake has any effect. Would be nice to test it.

Reply:
Unfortunately, I don't have any idea on how to write test for this change top of my head, any idea or suggestions?

Reply:
@darinadler - I tried to look into on how to write some test for it but I think it would be difficult at my novice level. Should I close this PR and leave comment on bugzilla for someone else to land this with testcase? Thanks!

Reply:
You can land it without a test case.

In the end if we have no idea what the effect is, it’s not so great; we could be making things worse somehow even.

Would be good to seek out someone who has an idea what effect an incorrect maximum would have. But that need not delay merging this corrected constant.

Reply:
If I am not mistaken @cdumez did this change in Blink based on the commit, I think he might have some insight. If he can share any insight, it would be great else I would land it. I agree with you that without doing proper testing, we are just trying to shoot in the blind.

Reply:
@Ahmad-S792 My change in WebKit had an associated (API) test. Your cherry-pick doesn't.

Reply:
Looks like Blink has API test coverage for Decimal while WebKit doesn't. Seems like we should introduce some Decimal API tests.

Reply:
What happened to `std::decimal`? Why can’t we use that?

Reply:
> What happened to `std::decimal`? Why can’t we use that?

Is this a thing? I found a proposal for std::decimal but it's not clear to me after some googling that it is part of the standard.